### PR TITLE
Implement New toolkit version csharp samples

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**
**Sample List-**
 
**1. incoming-webhook/csharp
2. meetings-app-icon-badging/chsharp
3. meetings-context-app/csharp
4. meetings-live-code-interview/csharp
5. msgext-ai-sentiment-analysis/csharp**

### CODE REVIEW

**Review by Alive-Fish — APPROVED** [review]
Only find the following issues. Please check them.

File: samples/meetings-context-app/csharp/TeamsApp/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿{
+  "profiles": {
+    // Debug project within Teams
+    "Microsoft Teams (browser)": {
+      "commandName": "Project",
+      "launchUrl": "https://teams.microsoft.com/l/app/%TEAMSAPPID%?installAppPackage=true&webjoin=true&appTenantId=%TENANTID%&login_hint=%USERNAME%"
```
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/meetings-context-app/csharp/TeamsApp/launchSettings.json:6]: Fixed

File: samples/meetings-live-code-interview/csharp/TeamsApp/launchSettings.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "profiles": {
+    // Debug project within Teams
+    "Microsoft Teams (browser)": {
+      "commandName": "Project",
+      "launchUrl": "https://teams.microsoft.com/l/app/%TEAMSAPPID%?installAppPackage=true&webjoin=true&appTenantId=%TENANTID%&login_hint=%USERNAME%"
```
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/meetings-live-code-interview/csharp/TeamsApp/launchSettings.json:6]: Fixed

File: samples/msgext-ai-sentiment-analysis/csharp/TeamsApp/launchSettings.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+﻿{
+  "profiles": {
+    // Debug project within Teams
+    "Microsoft Teams (browser)": {
+      "commandName": "Project",
+      "launchUrl": "https://teams.microsoft.com/l/app/%TEAMSAPPID%?installAppPackage=true&webjoin=true&appTenantId=%TENANTID%&login_hint=%USERNAME%"
```
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/msgext-ai-sentiment-analysis/csharp/TeamsApp/launchSettings.json:6]: Fixed

**Review by Alive-Fish — APPROVED** [review]

**Review by Mohammed-MSFT — APPROVED** [review]

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline -
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/11327771598

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline - 

https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/11327771598

**Review by Jegadeesh-MSFT — APPROVED** [review]
Looks good !

**Pull request merged**